### PR TITLE
Generate JS client code so we don't need a dynamic proxy

### PR DESCRIPTION
**Pull request by swankjesse**
This should also be useful when calling from JS into JVM.

### CODE REVIEW

File: ktbridge/src/jvmMain/kotlin/app/cash/quickjs/ktbridge/ktBridgeJvm.kt [hunk 1]
```diff
@@ -16,71 +16,10 @@
...
-      val globalName = "ktBridge_${nextGlobalId.getAndIncrement()}"
-      quickJs.evaluate("this.$globalName = $webpackModuleName.$packageName.$propertyName")
-      val internalBridge = quickJs.get(globalName, InternalBridge::class.java) as InternalBridge<T>
-      return internalBridge.toProxy(type, jsAdapter)
-    }
-  }
-}
-
-// TODO(jwilson): make private.
-fun <T : Any> InternalBridge<T>.toProxy(
```
Comment (swankjesse) [inline ktbridge/src/jvmMain/kotlin/app/cash/quickjs/ktbridge/ktBridgeJvm.kt:57]: No more dynamic proxies! RPC stubs on both ends are implemented with codegen.

File: ktbridge/plugin/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/OutboundCallRewriter.kt [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+ *         val outboundCall = OutboundCall(jsAdapter, internalBridge, "echo", 1)
+ *         outboundCall.parameter(request)
+ *         return outboundCall.invoke()
+ *       }
+ *     }
+ *   }
+ * }
+ * ```
+ */
+@ObsoleteDescriptorBasedAPI // TODO(jwilson): is there an alternative?
```
Comment (JakeWharton) [inline ktbridge/plugin/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/OutboundCallRewriter.kt]: Rewrite to owner types like you did the other? At least in a follow-up. Unless there's some other reason for keeping it for now...

**Review by JakeWharton — APPROVED** [review]

File: ktbridge/plugin/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/OutboundCallRewriter.kt (hunk 2 above)
Comment (swankjesse) [reply ktbridge/plugin/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/OutboundCallRewriter.kt]: Yeah, lemme do that right now.

Comment (swankjesse) [reply ktbridge/plugin/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/OutboundCallRewriter.kt]: Done.

**Pull request merged**